src/verifiers/documentStatus/signedProof/openAttestationSignedProof.ts

```diff
@@ -1,7 +1,7 @@
 import * as ethers from "ethers";
```

Comment:
do we delete this now that support has been removed from OA

Reply:
rmed